src/common/models/ICUHeadroom.ts

```diff
@@ -0,0 +1,230 @@
+import { nonNull } from 'common/utils';
+import {
+  CANActualsTimeseriesRow,
+  CANPredictionTimeseriesRow,
+  _Actuals,
+} from 'api/schema/CovidActNowStatesTimeseries';
+import { indexOfLastValue, hasRecentData, omitDataAfterDate } from './utils';
+
+// We sometimes need to override the ICU metric for locations due to bad data, etc.
+// TODO(https://trello.com/c/CPcYKmdo/): Review once we have better estimates for
+// these counties
+const ICU_HEADROOM_OVERRIDES = [
+  '01101', // Montgomery, AL
+  '48201', // Harris, TX
+];
+
+/**
+ * Default utilization to use (before decomp) if there isn't a location-specific
+ * value in the API.
+ */
+const ICU_DEFAULT_UTILIZATION = 0.75;
+
+/**
+ * We subtract this "decomp" factor from the typical ICU Utilization rates we
+ * get from the API to account for hospitals being able to decrease their utilization
+ * (by cancelling elective surgeries, using surge capacity, etc.).
+ */
+const ICU_DECOMP_FACTOR = 0.21;
+const ICU_DECOMP_FACTOR_STATE_OVERRIDES: { [key: string]: number } = {
+  Alabama: 0.15,
+  Arizona: 0.4,
```

Comment:
NOTE: I changed Arizona decomp from 0% to 40% to make it match up with the actuals from their state dashboard.  I think their capacity has changed or they've displaced non-covid patients or something.

With 0%, they were pegged at 100% since the beginning of their actuals (Apr 13).

Reply:
Oops.  I must have forgotten to push the commit where I tweaked it. It's 15% now.